components/eam/src/physics/crm/physpkg.F90

```diff
@@ -1562,24 +1567,149 @@ subroutine tphysbc(ztodt, fsns, fsnt, flns, flnt, &
+  type(physics_ptend)   :: ptend            ! indivdual parameterization tendencies
+  type(physics_ptend)   :: ptend_aero       ! ptend for microp_aero
+  type(physics_state)   :: state_alt        ! alt state for CRM input
+  integer  :: nstep                         ! current timestep number
+  real(r8) :: net_flx(pcols)
+  real(r8) :: cmfmc2(pcols,pverp)           ! Moist convection cloud mass flux
+  real(r8) :: dlf(pcols,pver)               ! Detraining cld H20 from shallow + deep convections
+  real(r8) :: dlf2(pcols,pver)              ! Detraining cld H20 from shallow convections
+  integer  :: lchnk                         ! chunk identifier
+  integer  :: ncol                          ! number of atmospheric columns
+  integer  :: ierr
+  integer  :: i,k,m,ihist                   ! Longitude, level, constituent indices
 
-  call check_energy_chng(state, tend, "crm_tend", nstep, ztodt,  &
-                         zero, mmf_qchk_prec_dp, mmf_qchk_snow_dp, mmf_rad_flux)
+  real(r8) :: sh_e_ed_ratio(pcols,pver)      ! shallow conv [ent/(ent+det)] ratio 
+
+  ! pbuf fields
+  integer itim_old
+  real(r8), pointer, dimension(:,:) :: cld  ! cloud fraction
+  real(r8), pointer, dimension(:,:) :: cldo ! old cloud fraction
+
+  ! energy checking variables
+  real(r8) :: zero(pcols)                    ! array of zeros
+  type(check_tracers_data):: tracerint       ! energy integrals and cummulative boundary fluxes
+  real(r8) :: zero_tracers(pcols,pcnst)
+
+  logical   :: lq(pcnst)
+
+  character(len=fieldname_len)   :: varname, vsuffix
+
+  !BSINGH - these were moved from zm_conv_intr because they are  used by aero_model_wetdep 
+  real(r8), dimension(pcols,pver) :: mu, eu, du, md, ed, dp
+  real(r8):: dsubcld(pcols) ! wg layer thickness in mbs (between upper/lower interface).
+  integer :: jt(pcols)      ! wg layer thickness in mbs between lcl and maxi.    
+  integer :: maxg(pcols)    ! wg top  level index of deep cumulus convection.
+  integer :: ideep(pcols)   ! wg gathered values of maxi.
+  integer :: lengath        ! w holds position of gathered points vs longitude index
+
+  logical :: l_tracer_aero, l_rad
+  
+  logical           :: use_ECPP
+  character(len=16) :: MMF_microphysics_scheme
+  real(r8), pointer :: mmf_clear_rh(:,:) ! CRM clear air relative humidity used for aerosol water uptake
+
+  ! ECPP variables
+  real(r8),pointer,dimension(:)   :: pblh              ! PBL height (for ECPP)
+  real(r8),pointer,dimension(:,:) :: acldy_cen_tbeg    ! cloud fraction
+  real(r8)                        :: dtstep_pp         ! ECPP time step (seconds)
+  integer                         :: necpp             ! number of GCM time steps in which ECPP is called once
+
+  call phys_getopts( l_tracer_aero_out      = l_tracer_aero      &
+                    ,l_rad_out              = l_rad              &
+                    ,use_ECPP_out           = use_ECPP           &
+                    ,MMF_microphysics_scheme_out   = MMF_microphysics_scheme )
```

Comment:
It does not look like this variable is being used in this routine.

Reply:
Yea, there's a chance we'll need to put it back for P3, but I'll remove it and we can worry about that later.